lib/matplotlib/cbook/__init__.py

```diff
@@ -2075,3 +2075,26 @@ def _check_and_log_subprocess(command, logger, **kwargs):
             .format(command, exc.output.decode('utf-8')))
     logger.debug(report)
     return report
+
+
+def _print_unit(st, dpi=None):
+    """
+    Convert from a string with a number and units into inches
+    """
+    print_units = {'cm': 2.54, 'pt': 72.0, 'mm': 25.4, 'in': 1.0}
+    if len(st) > 2:
```

Comment:
is that a leftover of supporting "2cm"?

Reply:
Not leftover yet.  I still think it’s an ok API to have both

Reply:
No problem, we can discuss this over the call.